### ISSUE TITLE
DART tree weights aren't visible in R

### ISSUE THREAD

**Issue by Kodiologist** [dmlc/xgboost#5051]
In particular, they don't appear in the output of `xgb.dump`.

**Comment by trivialfis** [comment]
@Kodiologist Is there any preferred schema for embedding this information into the text/json/dot dump format?

**Comment by Kodiologist** [comment]
Not really. I don't look at my trees most of the time. I just noticed this absence while I was debugging #5036.
